guides/common/modules/proc_configuring-provisioning-callback-for-a-host.adoc

```diff
@@ -47,8 +47,8 @@ For more information about job templates, see http://docs.ansible.com/automation
 |====
 |Name |Value |Description
 |`ansible_tower_provisioning` |true |Enables Provisioning Callback.
-|`ansible_tower_fqdn` |_controller.example.com_ |The fully qualified domain name (FQDN) of your {awx}.
-Do not add `https` because this is appended by {Project}.
+|`ansible_tower_api_url` |_https://controller.example.com/api/controller/v2_ |Defines the fully qualified domain name (FQDN) of your {awx} with the required API path.
+For {awx}{nbsp}2.4 or older, use `/api/v2` instead of `/api/controller/v2`.
```

Comment:
I think it's best to first submit this to `master` and then cherry pick it. Otherwise we'll have a regression on the next version.